<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryTest</class>
 <widget class="QMainWindow" name="EntryTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Admin/icon.qrc">
    <normaloff>:/icons/icons/backgrounds/hands.png</normaloff>:/icons/icons/backgrounds/hands.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#login_form{
background-color: rgb(48, 78, 151);
border-radius:40px;
margin: 60px;
}
QFrame#border_frame
{
background-color:#6fa844 ;
}
QFrame#main_frame{
background-image: url(:/icons/Background2.jpg);

background-position:center;
background-repeat:no-repeat;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="border_frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>40</number>
       </property>
       <property name="topMargin">
        <number>40</number>
       </property>
       <property name="rightMargin">
        <number>40</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item>
        <widget class="QFrame" name="main_frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="header_frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>120</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <layout class="QGridLayout" name="header_gridLayout">
               <item row="0" column="0" rowspan="2">
                <widget class="QLabel" name="hunar_logo">
                 <property name="minimumSize">
                  <size>
                   <width>70</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>70</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
background-image: url(:/icons/icons/backgrounds/Asset 7-100.jpg);
background-repeat: no-repeat; 
 background-position: right;
margin:5px;
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLabel" name="RAAS_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(48, 78, 151);
font: 63 32pt &quot;Bahnschrift SemiBold&quot;;</string>
                 </property>
                 <property name="text">
                  <string>RAAS</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLabel" name="std_name_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>130</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 14pt &quot;Bahnschrift SemiBold&quot;;
</string>
                 </property>
                 <property name="text">
                  <string>Student Name:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="3" colspan="2">
                <widget class="QLabel" name="disp_std_name_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(74, 74, 74);
font: 14pt &quot;Bahnschrift SemiBold&quot;;
</string>
                 </property>
                 <property name="text">
                  <string>Javeria Hassan</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QLabel" name="ET_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>541</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(48, 78, 151);
font: 25 22pt &quot;Bahnschrift Light&quot;;</string>
                 </property>
                 <property name="text">
                  <string>INSTITUTE OF EMERGING TECHNOLOGIES</string>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="2" colspan="2">
                <widget class="QLabel" name="time_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 14pt &quot;Bahnschrift SemiBold&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Time Remaining:</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="4">
                <widget class="QLabel" name="disp_time_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">
color: rgb(255, 0, 0);
font: 14pt &quot;Bahnschrift SemiBold&quot;;</string>
                 </property>
                 <property name="text">
                  <string>03:34</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="navigation_bar_frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>45</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="entry_test_label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>541</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(48, 78, 151);
font:20pt &quot;Bahnschrift Light&quot;;
color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.0340909 rgba(0, 0, 255, 255), stop:0.204545 rgba(255, 255, 255, 255), stop:0.318182 rgba(0, 170, 0, 255), stop:0.522727 rgba(255, 255, 255, 255), stop:0.727273 rgba(0, 0, 255, 255), stop:0.926136 rgba(0, 170, 0, 255));
</string>
               </property>
               <property name="text">
                <string>ENTRY TEST</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="quiz_frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <layout class="QGridLayout" name="quiz_gridLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="question_heading_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(48, 78, 151);
font: 18pt &quot;Bahnschrift SemiBold&quot;;</string>
                 </property>
                 <property name="text">
                  <string>QUESTION:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="1" rowspan="2">
                <widget class="QLabel" name="question_image_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>300</width>
                   <height>131</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(241, 241, 241);
background-image: url(:/icons/icons/white/image.svg);
background-repeat:no-repeat;
background-position:center;
font: 18pt &quot;Georgia&quot;;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="question_display_label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>550</width>
                   <height>100</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(48, 78, 151);
font:14pt &quot;Bahnschrift SemiBold&quot;;</string>
                 </property>
                 <property name="text">
                  <string>What is your city name?</string>
                 </property>
                 <property name="scaledContents">
                  <bool>true</bool>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QRadioButton" name="option1_radioButton">
                 <property name="minimumSize">
                  <size>
                   <width>550</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 14pt &quot;Bahnschrift SemiBold&quot;;
background-repeat:no-repeat;
background-position: center;
color: rgb(48, 78, 151);</string>
                 </property>
                 <property name="text">
                  <string>Karachi</string>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QRadioButton" name="option2_radioButton">
                 <property name="minimumSize">
                  <size>
                   <width>550</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 14pt &quot;Bahnschrift SemiBold&quot;;
background-repeat:no-repeat;
background-position: center;
color: rgb(48, 78, 151);</string>
                 </property>
                 <property name="text">
                  <string>Lahore</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QRadioButton" name="option3_radioButton">
                 <property name="minimumSize">
                  <size>
                   <width>550</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 14pt &quot;Bahnschrift SemiBold&quot;;
background-repeat:no-repeat;
background-position: center;
color: rgb(48, 78, 151);</string>
                 </property>
                 <property name="text">
                  <string>Islamabad</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QRadioButton" name="option4_radioButton">
                 <property name="minimumSize">
                  <size>
                   <width>550</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">font: 14pt &quot;Bahnschrift SemiBold&quot;;
background-repeat:no-repeat;
background-position: center;
color: rgb(48, 78, 151);</string>
                 </property>
                 <property name="text">
                  <string>Peshawer</string>
                 </property>
                </widget>
               </item>
               <item row="6" column="1">
                <widget class="QPushButton" name="next_btn">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>40</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color:  rgb(48, 78, 151);


font: 12pt &quot;Bahnschrift SemiBold&quot;;
border-radius: 20px;
color: rgb(241, 241, 241);
</string>
                 </property>
                 <property name="text">
                  <string>Next</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../Admin/icon.qrc"/>
 </resources>
 <connections/>
</ui>
